app/jobs/event_job.rb

```diff
@@ -5,14 +11,52 @@ class EventJob
   include Hydra::AccessControlsEnforcement
   include SufiaHelper
 
+  # queue to runthe job on
   def queue_name
     :event
   end
 
-  attr_accessor :generic_file_id, :depositor_id
+  attr_accessor :depositor_id
 
-  def initialize(generic_file_id, depositor_id)
-    self.generic_file_id = generic_file_id
+  # @param the id of the user to create the event for
+  def initialize(depositor_id)
     self.depositor_id = depositor_id
   end
+
+  # Method used to cause the event to be created
+  def run
+    # Log the event to the depositor's profile stream
+    log_user_event
+
+    # Fan out the event to all followers who have access
+    log_to_followers
+  end
+
+  # override to provide your specific action for the event you are logging
+  # @abstract
+  def action
+    raise(NotImplementedError, "#action should be implmented by an child class of EventJob")
```

Comment:
Now that is @awead's typo :smile: I copied his change...  Most of them are mine so it is good to see other typoing too!